notebooks/kbmod_search_results_for_fakes.ipynb

```diff
@@ -0,0 +1,287 @@
+    "import astropy.units as u\n",
+    "from astropy.table import Table\n",
+    "\n",
+    "# Data paths\n",
+    "wu_path = \"/epyc/projects/kbmod/runs/wbeebe/1000_imgs_5_16/reprojected_wu.fits\"  # A reflex-corrected WorkUnit\n",
+    "res_path = \"/epyc/projects/kbmod/runs/wbeebe/1000_imgs_5_16_slow\"\n",
+    "# Path to known fakes (with reflex-corrected) coordinates on the dates used in this KBMOD search.\n",
+    "fakes_path = \"/epyc/projects/kbmod/runs/wbeebe/fakes_detections_20190404_20190505_simple.csv\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reload a Saved KBMOD WorkUnit\n",
+    "Note that this WorkUnit was reflex-corrected with a guess distance of 40 AU."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wu = WorkUnit.from_fits(wu_path)\n",
+    "stack = wu.im_stack"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load the KBMOD Results and Known Fakes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = Results.read_table(os.path.join(res_path, \"results.ecsv\"))\n",
+    "print(f\"Loaded {len(results)} results.\")\n",
+    "results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And now load our known fakes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fakes = Table.read(fakes_path, format=\"csv\")\n",
+    "fakes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Search for results that are near known fakes\n",
+    "\n",
+    "astropy allows us to take two catalogus of coordinates (represented by `SkyCoord` objects) and easily search for nearest neighbors between them. \n",
+    "\n",
+    "First, as a simple approximation let's translate the initial (x, y) of each of our results into an (ra, dec). Note that we are using a reflex-corrected WCS from our `WorkUnit` so the (ra, dec) will be in reflex-corrected space."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results_ra_dec = [wu.wcs.pixel_to_world(r[\"x\"], r[\"y\"]) for r in results]\n",
+    "results.table[\"ra_dec\"] = results_ra_dec\n",
+    "results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can translate our (ra, dec) pairs into single `SkyCoord` objects.\n",
+    "\n",
+    "Then we can use astopy's `search_around_sky` to find which KBMOD results are near our known fakes, with a max separation limit of 1 arcsecond"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result_coords = SkyCoord(\n",
```

Comment:
I think you can just use `results.table["ra_dec"]` here. So the whole thing could be:
```
results.table["ra_dec"] = wu.wcs.pixel_to_world(results["x"], results["y"])
... load the fakes ...
idx1, idx2, sep2dAngle, dist3d = search_around_sky(results.table["ra_dec"], fake_coords, 1 * u.arcsecond)
```